baseplate/server/healthcheck.py

```diff
@@ -0,0 +1,74 @@
+"""Check health of a baseplate service on localhost."""
```

Comment:
totally valid. don't ever hold back on asking for docs/explanation! :)

i've added another file to the docs explaining this tool, does that help?